src/Components/App/App.tsx

```diff
@@ -45,11 +49,27 @@ class App extends Component<{}, IAppState> {
       })
   }
 
+  findPostsWithSearchTerm = (searchTerm: string) => {
+    console.log("SEARCH TERM: ",searchTerm)
+
+    // this is only finding the first one, not sure why
+    return this.setState({ foundPosts: this.state.allPosts.filter(post => post.content.includes(searchTerm))} )
+
+  }
```

Comment:
This is where I'm filtering, and it's only returning the first matching post. I'm only doing content right now, just to work out filtering and I'll add the post.title later, as well as maybe the author so we can also search by author.

Reply:
This filter works great on my end!

Reply:
Wow!  That's weird and also great.  🤪